### ISSUE TITLE
Climates of Ferngill & Lunar Disturbances: Blood Moon flickers

### ISSUE THREAD

**Issue by JohnsonNicholas** [JohnsonNicholas/SDVMods#60]
User reports flickering as if something is trying to override it.

**Comment by JohnsonNicholas** [comment]
Needs:

- [ ] Updating Lunar Disturbances to use Climates API to check for certain weathers
- [ ] Updating Climates to add the Blood Moon as a weather and update special effects

**Comment by JohnsonNicholas** [comment]
Check weathers for blood moon usage, look into if I need to do anything with Climates own weather parser.
Original report: 
"Hi! I'm using this mod alongside Climates and Dynamic Nighttime. I turned on the blood moon effect and since I did that, the weather icon keeps flashing in the spring 7, year 1 files I had. I think it's trying to display the blood moon icon because I can catch a glimpse of something dark and orange, but the Climates mod seems to keep overriding it and displaying the current day weather. Two files so far have done this and both have had non-sunny weather that day. One was stormy, and the other was windy. The message of it being a full moon didn't display in either. The second file had the flashing, but when the day ended, the moon phase shown was a full moon, rather than a blood moon. Both files were created and advanced in before I turned on the blood moon. Day 8 had no graphic issues, even in files that hadn't been opened before switching on lunar disturbances."
